packages/types/src/discordeno.ts

```diff
@@ -148,6 +148,8 @@ export interface ButtonComponent {
     /** Whether this emoji is animated */
     animated?: boolean
   }
+  /** Identifier for a purchasable SKU, only available when using premium-style buttons */
+  skuId?: bigint
```

Comment:
BigString would be more appropriate as then it also supports strings
```suggestion
  skuId?: BigString
```